### ISSUE TITLE
Mistake in the alt text for the diagram in "Problem 199: Iterative Circle Packing"

### ISSUE THREAD

**Issue by sidemt** [freeCodeCamp/freeCodeCamp#44245]
In the challenge below, the alt text for the diagram is set to "a diagram of non-overlapping concentric circles".
But if my understanding is correct, they are not "concentric circles" (circles which share the same center).

## Affected page

Coding Interview Prep > Project Euler > Problem 199: Iterative Circle Packing
https://www.freecodecamp.org/learn/coding-interview-prep/project-euler/problem-199-iterative-circle-packing

## Expected behavior

The alt text should be changed to "a diagram of non-overlapping circles" or something.
(I'm not a native English speaker, so please fix it if there is a better description.)

## Screenshots

![project_euler_199_issue](https://user-images.githubusercontent.com/25644062/142926610-bbf460e8-8320-45e2-808a-e1de401e74f3.png)

## System

- Device: Laptop
- OS: Windows 10
- Browser: chrome
- Version: 96.0.4664.45

## Additional context

Originally reported in a comment on Crowdin duing translation.
https://freecodecamp.crowdin.com/translate/690087755819f2f67b26dbee3be0eb53/22524/en-ja/132?filter=basic&value=7#q=662762

**Comment by gikf** [comment]
Thanks for opening this issue.

This looks something that can be fixed by "first time" code contributors to this repository. Here are the files that you should be looking at to work on a fix:

List of files:

1. https://github.com/freeCodeCamp/freeCodeCamp/blob/main/curriculum/challenges/english/10-coding-interview-prep/project-euler/problem-199-iterative-circle-packing.md

Please make sure you read [our guidelines for contributing](https://contribute.freecodecamp.org/#/), we prioritize contributors following the instructions in our guides. Join us in [our chat room](https://chat.freecodecamp.org/channel/contributors) or [the forum](https://forum.freecodecamp.org/c/contributors/3) if you need help contributing, our moderators will guide you through this.

Sometimes we may get more than one pull requests. We typically accept the most quality contribution followed by the one that is made first.

Happy contributing.

**Comment by asadrjhk** [comment]
@gikf I would like to work on this.

**Comment by gikf** [comment]
Hey @asadrjhk

Issues are typically not assigned to a specific person, other than long-time contributors. You can read more about this at https://contribute.freecodecamp.org/#/FAQ?id=how-can-i-get-an-issue-assigned-to-me. 

Issues labelled with `help wanted` or `first timers only` are open for contributions for all.

Please make sure you read [our guidelines for contributing](https://contribute.freecodecamp.org/#/). Join us in [our chat room](https://chat.freecodecamp.org/channel/contributors) or [the forum](https://forum.freecodecamp.org/c/contributors/3) if you need help contributing, our moderators will guide you through this.

**Comment by asadrjhk** [comment]
@gikf sorry about that, as this  was my first time in freeCodeCamp, I'll make sure to read the contribution guidelines
